tty/uart16550/uart16550.c

```diff
@@ -165,11 +165,11 @@ uint8_t uart_get(oid_t *oid)
 {
 	unsigned int i;
 
-	for (i = 0; i < sizeof(uarts) / sizeof(uart_t); i++) {
+	for (i = 0; i < sizeof(uarts) / sizeof(uart_t *); i++) {
```

Comment:
Perhaps sizeof(uarts) / sizeof(uarts[0])?

Reply:
This part proposed by kemonats is good. I commented these returns. He changed first return with break and proposed to changed the return 0 located at the function end with return i. This change will lead us to incorrect behavior when wrong oid will be passed in WRITE and READ messages.